README.md

```diff
@@ -38,12 +38,12 @@ sudo apt update
 ./install.sh --update
 ```
 
-Use `./{game}/install-{game}.sh` to update only one game.
+Use `./{game}/install.sh` to update only one game.
 
 ### Game servers management
 
 ```sh
-sudo systemctl {start, stop or restart} {game}-server.service
+sudo service {game}-server {start|stop|restart|status}
```

Comment:
I think keeping both `service` and `systemctl` would be even better.

```suggestion
sudo service {game}-server {start|stop|restart|status}
# or
sudo systemctl {start|stop|restart|status} {game}-server
```

Reply:
Updated accordingly.